src/erl_optics.erl

```diff
@@ -198,6 +203,19 @@ start_optics(Prefix, Lenses) ->
     },
     supervisor:start_child(erl_optics_sup, Poller).
 
+start_optics(Prefix) ->
```

Comment:
What if this function called `start_optics(Prefix, make_lenses_list())` instead?